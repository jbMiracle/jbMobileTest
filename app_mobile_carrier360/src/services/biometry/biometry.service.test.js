import { queryBiometricSupport, authenticate } from '.';

describe('queryBiometricSupport()', async () => {
  it('should return an object with isSupported and isSetup', async () => {
    const biometricSupport = await queryBiometricSupport();
    expect(biometricSupport).toEqual({
      isSupported: false,
      supportedType: null,
    });
    expect(biometricSupport).toMatchSnapshot();
  });
});
describe('authenticate()', () => {
  it('should return, whether or not the user authenticated successfully, with a boolean', async () => {
    const authenticating = await authenticate();
    expect(authenticating).toBe(false);
    expect(authenticating).toMatchSnapshot();
  });
});
