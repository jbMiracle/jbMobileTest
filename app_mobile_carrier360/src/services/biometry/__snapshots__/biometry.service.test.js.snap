// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authenticate() should return, whether or not the user authenticated successfully, with a boolean 1`] = `false`;

exports[`queryBiometricSupport() should return an object with isSupported and isSetup 1`] = `
Object {
  "isSupported": false,
  "supportedType": null,
}
`;
